extra/modules/confiant-ad-quality/src/main/java/org/prebid/server/hooks/modules/com/confiant/adquality/core/RedisClient.java

```diff
@@ -0,0 +1,84 @@
+package org.prebid.server.hooks.modules.com.confiant.adquality.core;
+
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import org.prebid.server.hooks.modules.com.confiant.adquality.model.BidScanResult;
+import org.prebid.server.hooks.modules.com.confiant.adquality.model.OperationResult;
+import org.prebid.server.hooks.modules.com.confiant.adquality.model.RedisBidsData;
+import redis.clients.jedis.DefaultJedisClientConfig;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.exceptions.JedisException;
+
+import java.util.Collections;
+import java.util.List;
+
+public class RedisClient {
+
+    private static final Logger logger = LoggerFactory.getLogger(RedisClient.class);
+
+    private final RedisParser redisParser = new RedisParser();
+
+    private final String apiKey;
+
+    private Jedis jedis;
+
+    public RedisClient(
+            String apiKey,
+            String host,
+            int port,
+            String password
+    ) {
+        this.apiKey = apiKey;
+
+        try {
+            this.jedis = createJedis(host, port, password);
+        } catch (JedisException e) {
+            logger.info("Can't establish Redis connection {0}", e.getMessage());
+        }
+    }
+
+    private Jedis createJedis(String host, int port, String password) {
+        DefaultJedisClientConfig config = DefaultJedisClientConfig
+                .builder()
+                .password(password)
+                .build();
+
+        return new Jedis(host, port, config);
+    }
+
+    public BidsScanResult submitBids(RedisBidsData bids) {
+        if (jedis != null && bids.getBresps().size() > 0) {
+            try {
+                final String submitHash = jedis.get("function_submit_bids");
```

Comment:
Why don't you provide function code? Also, what is the point in using redis here? This has to be a backend application.